src/components/PriceCheckForm.js

```diff
@@ -1,33 +1,70 @@
 import React, { Component } from 'react'
+import axios from 'axios';
+import { categories } from '../utils/Constants.js';
+
 
 class PriceCheckForm extends Component {
     constructor(props){
         super(props)
         this.state = {
-            postcode: ""
+            postcode: "",
+            selectedCategory: "",
+            searchString: ""
         }
 
         this.handleSubmit = this.handleSubmit.bind(this);
-        this.handleChange = this.handleChange.bind(this);
-       
+        this.handlePostcodeChange = this.handlePostcodeChange.bind(this);
+        this.handleCategoryChange = this.handleCategoryChange.bind(this);
+        this.handleSearchStringChange = this.handleSearchStringChange.bind(this);
+        this.fetchSuppliers = this.fetchSuppliers.bind(this)
```

Comment:
You don’t need to bind this function, the arrow function declaration does that for you - but maybe consider using all arrow functions or all old school functions to keep things in the same format